applications/box.md

```diff
@@ -0,0 +1,213 @@
+# Crust Grant Proposal
+
+* **Project Name:** box
+* **Team Name:** Functionland
+* **Payment Address:** 0x42DD4709a336B8cE54b873069A5c09Ed27ec9Dd8
+* **Receivable Token:** USDT
+
+> ⚠️ *The combination of your GitHub account submitting the application and the payment address above will be your unique identifier during the program. Please keep them safe.*
+
+> ⚠️ *When you submit an application for a Grant，You confirm that you have read these Terms and Conditions carefully and that you agree to be bound by them.*
+
+### Overview
+
+Please provide the following:
+
+  * A project name: box
+  
+  * A brief description of your project: 
+  
+  The mainstream developer experience in the world of independent front-end developers is to purchase a managed service provided by a cloud vendor, which abstracts away the complexity of dealing with back-end development and enable front-end developers to focus on end-to-end user experience for their clients. Currently, there is no option for a front-end developer to build on top of IPFS with a "Developer Experience" similar to serverless functions. They need to run a full js-ipfs node on the platform they want to develop on, for example on Android, which has different file system access than iOS or other operating systems. "box" is an IPFS node which aims to separate the server from front-end development and provides front-end APIs for developers to connect to, using Libp2p, giving them the same experience as a cloud server for any "box" node. To give a full application server experience to the developer, two protocols are being developed in "box" for a start:
+
+File Protocol
+
+Data Protocol
+
+Front-end developers work with a "box" like a server without the need to run a full IPFS node on the platform they develop for, and interface with the "box" node to store or fetch data using libp2p. End users, their clients, can treat "box" like a server to keeps their data safe and gives them all functionalities of IPFS like cluster synchronization. If they lose their phone with the application installed, their files are safe in the "box" node.
+  
+  * An indication of how your project relates to / integrates into Crust: We will use crust as a pinning service to allow extra level of resiliance for those who want to have their files pinned out of their box
```

Comment:
```suggestion
  * An indication of how your project relates to / integrates into Crust: We will use crust as a pinning service to allow extra level of resiliance for those who want to have their files pinned out of their box. Crust guarantees high availability of pinned data across IPFS network and this is exactly the advantage that we hope to utilize in this proposal. When sharing between users there is no way other than having them pinned on a service like crust, however while the photo or data is not shared with anyone and user is within the local network(e.g. at home) we leave this choice to user if they want it pinned or not as we want to enable them run it at zero cost if they choose to, and we leave it to user if they want to disconnect a box from network for any privacy reasons.
```

Reply:
This is also response to: https://github.com/crustio/Crust-Grants-Program/pull/39/files#r669248392